runtime/wasm/src/module.rs

```diff
@@ -299,6 +305,32 @@ where
         Ok(None)
     }
 
+    /// function store.get(entity: string, id: string): void
+    fn store_get(
+        &self,
+        entity_ptr: AscPtr<AscString>,
+        id_ptr: AscPtr<AscString>,
+    ) -> Result<Option<RuntimeValue>, Trap> {
+        let store_key = StoreKey {
+            subgraph: self.subgraph.id.clone(),
+            entity: self.heap.asc_get(entity_ptr),
+            id: self.heap.asc_get(id_ptr),
+        };
+
+        // Retrieve an Entity from the store
+        let get_result = self
+            .store
+            .lock()
+            .unwrap()
+            .get(store_key)
+            .map(|entity| entity);
+        let get_result: HashMap<_, _> = match get_result {
+            Ok(entity) => entity.into(),
```

Comment:
Type ascription is still experimental; other ideas for supplying into() with the type without using an intermediate variable?

Reply:
I prefer `from` over `into` to avoid these problems, so you can do `HashMap::from(entity)`